api/src/modules/Session/model.ts

```diff
@@ -115,19 +124,20 @@ export function getSessionCrud(prisma: PrismaClient) {
     createSignInSession: async (userId: string) => generateTokenAndSession(
       prisma,
       userId,
-      { expiryDurationSeconds: ms(JWT_EXPIRATION_PERIOD) / 1000, name: 'Sign in' },
+      { expiryDurationSeconds: ms(JWT_EXPIRATION_PERIOD) / 1000, name: 'Sign in', originRestriction: '*' },
```

Comment:
I think sign up and sign in tokens should be limited to the origin they've sent from. For example, sign up done via our ui should display `https://main.switchboard.k8s.sidestream.tech`

Reply:
added the env var for it, readme row and kube values adjustment

Reply:
> added the env var for it

As specified, "limited to the origin they've _sent from_": ie, a frontend making the sign in request will automatically provide `https://main.switchboard.k8s.sidestream.tech` inside origin, without any additional env variables required. If the origin is not set in the request, you can fallback to `*`

Reply:
sorry for missing this one.